workbench/config/defaults.php

```diff
@@ -321,7 +322,7 @@
     $config["defaultApiVersion"]  = array(
         "label" => "Default API Version",
         "description" => "Default API version to be used for login. This setting does not affect the API version of the current session. Recommended to choose latest version. Some features may act unexpectedly when using older versions.",
-        "default" => "46.0",
+        "default" => "48.0",
```

Comment:
default version should be one behind the current version. So this should be "47.0"